ui/react/configuration/reducers.js

```diff
@@ -0,0 +1,246 @@
+import * as actionTypes from './actionTypes';
+import immutable from 'immutable';
+
+const defaultUiState = {
+    nomenclatures: {
+        itemName: [
+            'country',
+            'channel',
+            'region',
+            'city',
+            'organization',
+            'role',
```

Comment:
Please note the pull request is targeted for ci_updateVersionsToUtFrontAndUtFrontReact